pkg/apis/pipelines/v1alpha1/gitopsservice_types.go

```diff
@@ -6,6 +6,7 @@ import (
 
 // GitopsServiceSpec defines the desired state of GitopsService
 type GitopsServiceSpec struct {
+	SSO bool `json:"sso,omitempty"`
```

Comment:
Consider changing the field name to more descriptive name like `KeycloakEnabled`, `SSOEnabled`.   SSO is actually a pretty board term.

Reply:
Changed the Name from SSO to EnableSSO

Reply:
That's probably fine.  My thought is that SSO is a board term.   SSO can mean OIDC/SAML/AD.   Our SSO here only means KeyCloak or OIDC.